tools/sigclient/cmd/sigclient.go

```diff
@@ -76,8 +88,11 @@ var esBulkCmd = &cobra.Command{
 		log.Infof("bearerToken : %+v\n", bearerToken)
 		log.Infof("generatorType : %+v. Add timestamp: %+v\n", generatorType, ts)
 		log.Infof("eventsPerDay : %+v\n", eventsPerDay)
+		if enableVariableNumColumns {
```

Comment:
nit we should keep the format similar to other prints we have earlier. 
if they are not set , then should display the default value